set.mm

```diff
@@ -474,18 +474,24 @@ The Metamath (Metamath-exe) program Proof Assistant requires proofs to
        point in a proof to allow an independent subproof to be developed on the
        side, for later use as part of the final proof.
 
-       _Instructions_:  (1) Assign this inference to any unknown step in the
-       proof.  Typically, the last unknown step is the most convenient, since
-       "MM-PA> ASSIGN LAST" can be used.  This step will be replicated in
-       hypothesis a1ii.1, from where the development of the main proof can
-       continue.  (2) Develop the independent subproof backwards from
-       hypothesis a1ii.2.  If desired, use a "MM-PA> LET STEP" command to
-       pre-assign the conclusion of the independent subproof to a1ii.2.  (3)
-       After the independent subproof is complete, use "MM-PA> IMPROVE ALL" to
-       assign it automatically to an unknown step in the main proof that
-       matches it.  (4) After the entire proof is complete, use "MM-PA>
-       MINIMIZE__WITH *" to clean up (discard) all ~ a1ii references
-       automatically.
+       _Instructions_:
+       <HTML><ol><li>Assign this inference to any unknown step in the proof.
+       Typically, the last unknown step is the most convenient, since
+       <code>MM-PA> ASSIGN LAST</code> can be used.  This step will be
+       replicated in hypothesis a1ii.1, from where the development of the main
+       proof can continue.</li><li>Develop the independent subproof backwards
+       from hypothesis a1ii.2.  If desired, use a <code>MM-PA> LET STEP</code>
+       command to pre-assign the conclusion of the independent subproof to
+       a1ii.2.</li><li>After the independent subproof is complete, use
+       <code>MM-PA> IMPROVE ALL</code> to assign it automatically to an unknown
+       step in the main proof that matches it.</li><li>After the entire proof
+       is complete, use <code>MM-PA> MINIMIZE_WITH *</code> to clean up
+       (discard) all ~ a1ii references automatically.</ol></HTML>
+
+       This can also be used to apply subproofs from other theorems.  In step
+       2, simply assign the theorem to a1ii.2, and run
+       <HTML><code>MM-PA> EXPAND <theorem></code></HTML> to "import" a subproof
+       from another theorem.
```

Comment:
This looks prettier indeed, and this change is good to me.  As a loose principle, I think there are two things to be wary of:
* do not spend too much time on rich formatting of comments since the further we depart from plain text, the harder the database is to maintain (but of course, when necessary for improving understanding, do use available functionalities);
* if there are general instructions or remarks about "proof engineering" which are not specific to set.mm, they probably have a better place to be that in set.mm.

Again, I think that in the current case this is ok (even though the method with using a1ii is generalizable to various sorts of databases, and these general guidelines could be added somewhere more general).